_includes/components/footer.html

```diff
@@ -29,23 +29,28 @@
   </div>
   {% endif %}
 
-  {% if footer.contact %}
```

Comment:
I think there may actually be some utility in keeping a conditional statement wrapped around this entire block just in the off chance that the project doesn't include neither logos or contact information in the footer. In that case we would just be left with an empty bar since I think there is some margin/padding associated with the outer `usa-footer-secondary_section` class.

Maybe instead we could extend this conditional to include a check for logos as well? Something like `{% if footer.contact or footer.logos %}` - thoughts?

Reply:
I like this approach accounting for both footer logos and contact info! 🎈

Reply:
Good call! I added the check for contact, logos, and headers since any of them could appear independently.